src/client.js

```diff
@@ -76,6 +76,10 @@ function Client(manager, name, config) {
 		id: id++,
 		name: name,
 		networks: [],
+		push: { // TODO: This should not ship
```

Comment:
It's here until we figure out push limits (e.g. how many messages per minute and stuff like that.)